chessboard/index.js

```diff
@@ -0,0 +1,45 @@
+
+
+const container = document.getElementById("chess-container");
+
+const createBoard = () => {
```

Comment:
You can use createDocumentFragment to create the board entirely in memory before appending it to the DOM, which is more efficient, especially for larger numbers of elements. Directly manipulating the DOM can be avoided here 

```
const container = document.getElementById("chess-container");

const createBoard = (container) => {
    const fragment = document.createDocumentFragment();

    for (let row = 0; row < 8; row++) {
        const rowDiv = document.createElement('div');
        for (let col = 0; col < 8; col++) {
            const cellSpan = document.createElement('span');
            cellSpan.id = `${row * 8 + col}`;
            rowDiv.appendChild(cellSpan);
        }
        fragment.appendChild(rowDiv);
    }

    container.appendChild(fragment);
}


createBoard(container);
```